packages/paste-website/src/pages/components/disclosure/index.mdx

```diff
@@ -0,0 +1,499 @@
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure variant="contained">
+  <DisclosureHeading as="h2" variant="heading60">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+### Controlling initial state
+
+You can control the intial state of the disclosure to be expanded or collapsed by setting the `visible` prop.
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure visible>
+  <DisclosureHeading as="h2" variant="heading30">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+## States
+
+### Disabled
+
+Each Disclosure can be disabled by setting the `disabled` prop on the `DisclosureHeading`
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+  }}
+>
+  {`<Disclosure visible>
+  <DisclosureHeading as="h2" disabled variant="heading30">
+    Disclosure Heading
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+## Composition notes
+
+Both `DisclosureHeading` and `DisclosureContent` accept any child component, leaving flexibility to create a large number of custom compositions for your product needs.
+
+The only thing you **must not** do is place a focusable element inside the `DisclosureHeading`, see [accessibility notes](/components/disclosure#accessibility).
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+    Text,
+  }}
+>
+  {`<Disclosure>
+  <DisclosureHeading as="h2" variant="heading30">
+    Disclosure Heading <Text color="colorTextWeak" fontSize="fontSize50" as="span">(3 new items)</Text>
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+<LivePreview
+  scope={{
+    Disclosure,
+    DisclosureHeading,
+    DisclosureContent,
+    ErrorIcon,
+    MediaObject,
+    MediaFigure,
+    MediaBody,
+    Text,
+  }}
+>
+  {`<Disclosure variant="contained" visible>
+  <DisclosureHeading as="h2" variant="heading30">
+    <MediaObject verticalAlign="center">
+      <MediaBody>
+        Disclosure Heading <Text color="colorTextWeak" fontSize="fontSize50" as="span">- deployment issue</Text>
+      </MediaBody>
+      <MediaFigure>
+        <ErrorIcon iconColor="colorTextError" decorative={false} title="Error" />
+      </MediaFigure>
+    </MediaObject>
+  </DisclosureHeading>
+  <DisclosureContent>
+    Content
+  </DisclosureContent>
+</Disclosure>`}
+</LivePreview>
+
+## Anatomy
+
+### Disclosure
+
+#### Contained
+
+Same Anatomy as [Card](/components/card).
+
+### DisclosureHeading
+
+Same Anatomy as [Heading](/components/heading), plus:
```

Comment:
So, we don't have the heading anatomy documented here. File a ticket for it?

Reply:
Separate Ticket